runner.py

```diff
@@ -0,0 +1,51 @@
+import runner_run_funcs
+import json
+
+
+class Runner:
+
+    run_func_ltable, data_gen_func_ltable = runner_run_funcs.get_run_funcs()
+
+    def __init__(self, param_json, sim_type, data_gen_type):
+        self._params = json.loads(param_json)
+        self._run(sim_type, data_gen_type)
```

Comment:
You should probably rename `_run` to `_init` or just inline the code into the `__init__` method.